playwright-tests/utils/login.ts

```diff
@@ -0,0 +1,33 @@
+import { Page } from '@playwright/test';
+import { confirmPem } from './confirmPem';
+import { DataTestIdsEnum } from '../../src/localConstants/dataTestIds.enum';
+import { LoginFilesEnum } from '../utils/enums.ts';
+
+interface Login {
```

Comment:
This needs a `Type` suffix or something indicating it's a type, it's too vague like this.